### ISSUE TITLE
vertx:debug - To make java remote debugging easy

### ISSUE THREAD

**Issue by davsclaus** [fabric8io/vertx-maven-plugin#67]
It would be nice if you can start vert.x in Java debug mode, so you can just attach a remote debugger on port 5005

**Comment by cescoffier** [comment]
I was thinking the same this morning. The only difference, I was using the port 5000. 

One thing to notice is that in redeploy mode, you need to re-attach your debugger after every redeployment.

**Comment by davsclaus** [comment]
5005 is the default remote debug port used by many IDEs etc.

**Comment by davsclaus** [comment]
Also the good trick with mvDebug vertx:run does not work. You can connect to remote port 8000 (mvnDebug uses 8000) but the break points never gets activated in your Vert.X application code.

So a specialized vertx:debug goal would be awesome, also its easier to use, than the mvnDebug trick

**Comment by cescoffier** [comment]
Definitely, 

My approach right now was:

```
vertx:run -Dvertx.debug=true -Dvertx.debug.suspend=true|false -Dvertx.debug.port=5005
```

Suspend is set to false by default, However, I'm not sure it is the right default. 
For the port, let's use 5005 by default.

But I agree, a mojo would simplify the command line to:

```
vertx:debug -Ddebug.suspend=true|fase -Ddebug.port=5005.
```

**Comment by cescoffier** [comment]
Fixed in #68
